Adapter/StorageAdapter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Adapter
+{
+    class StorageAdapter : IStorage
+    {
+        private readonly Storage storage;
+
+        public StorageAdapter()
+        {
+            storage = new Storage();
+        }
+
+        public void GetFormattedItem()
+        {
+            Console.WriteLine("--------------");
```

Comment:
It's not decorator, no need to decorate the output

Reply:
done